src/ImageSharp/Processing/Extensions/LightnessExtension.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+using SixLabors.ImageSharp.Processing.Processors.Filters;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp.Processing
+{
+    /// <summary>
+    /// Defines extensions that allow the alteration of the hue component of an <see cref="Image"/>
+    /// using Mutate/Clone.
+    /// </summary>
+    public static class LightnessExtension
+    {
+        /// <summary>
+        /// Alters the hue component of the image.
+        /// </summary>
+        /// <param name="source">The image this method extends.</param>
+        /// <param name="lightness">Lightness parameter of image in HSL color scheme.</param>
```

Comment:
I'm not sure this is an accurate description. 

In HSL lightness corresponds to the arithmetic mean of the largest and smallest RGB components, In a color matrix the W(hiteness) in RGBAW is basically added to multipliers of each RGB component.

Reply:
![image](https://user-images.githubusercontent.com/3886308/65041210-e09c3e80-d95e-11e9-9175-403b4c40eb64.png)

Reply:
What I mean is that we are not operating within the HSL colorspace. HSL lightness would likely require additional calculation.

Lightness can mean many different things. The algorithm I supplied you gave a non-weighted alteration to the white value of RGBA. 

Take a look at this article to help you understand the difference.
https://thoughtbot.com/blog/closer-look-color-lightness

Reply:
Ok. I'll remove all hsl references.